contrib/apache-httpd/mod_arpa2_tlspool.c

```diff
@@ -0,0 +1,314 @@
+/* Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ap_config.h"
+#include "ap_mmn.h"
+#include "httpd.h"
+#include "http_config.h"
+#include "http_connection.h"
+#include "http_core.h"
+#include "http_log.h"
+#include "http_vhost.h"
+#include "apr_strings.h"
+#include "apr_portable.h"
+#include <tlspool/starttls.h>
+
+module AP_MODULE_DECLARE_DATA tlspool_module;
+
+#define strcEQ(s1,s2)    (strcasecmp(s1,s2)    == 0)
+
+/*
+ *  the table of configuration directives we provide
+ */
```

Comment:
This table is far from complete!  Please check with the document for the others and supply them along with this?